extensions/amp-story/1.0/amp-story.css

```diff
@@ -984,3 +984,12 @@ amp-story-page .i-amphtml-story-page-open-attachment-label {
   text-shadow: 0px 0px 6px rgba(0, 0, 0, 0.36) !important;
   white-space: nowrap !important;
 }
+
+amp-story-quiz {
+  align-self: flex-start;
```

Comment:
Sorry I missed this, why not center?

Reply:
Ah this was a hold-over from when I was testing why the sizing changed, changing back to center